src/main/java/me/oyurimatheus/nossoservicodepagamento/payment/PaymentMethod.java

```diff
@@ -0,0 +1,66 @@
+package me.oyurimatheus.nossoservicodepagamento.payment;
+
+enum PaymentMethod {
+
+    CREDIT_CARD {
+
+        @Override
+        String description() {
```

Comment:
Pq isso não é atributo?

Reply:
Porque como eu modelei usando um enum, achei que podia deixar como um método mesmo...

Reply:
Mas realmente, se for um atributo o código fica mais enxuto